requirements/base.in

```diff
@@ -53,6 +53,9 @@ pymongo
 dict2xml
 lxml
 
+# python 3 compatibility
+future
```

Comment:
Can we please remove this requirement, I think it can come in the `python-2-3` branch, the upstream package that required this https://github.com/onaio/python-json2xlsclient/commit/dc545c7154db000ca283a57fd39af7320ee9beea handles it and as such it is not necessary to have this here.